rfcs/0180-broken-package-removal.md

```diff
@@ -0,0 +1,101 @@
+---
+feature: broken-package-removal
+start-date: 2024-07-14
+author: Jörg "Mic92" Thalheim
+co-authors:
+shepherd-team:
+shepherd-leader:
```

Comment:
RFCSC:

```suggestion
shepherd-team: @emilazy @preisi @jopejoe1 
shepherd-leader: @jopejoe1 
```

Reply:
@jopejoe1 we tentatively made you shepherd-leader. If you accept please follow the steps described [here](https://github.com/NixOS/rfcs?tab=readme-ov-file#shepherd-leader)

Reply:
Yes, I do accept.